plugin/pkg/shoot/quotavalidator/admission.go

```diff
@@ -332,7 +333,12 @@ func (q *QuotaValidator) findShootsReferQuota(quota core.Quota, shoot core.Shoot
 	return shootsReferQuota, nil
 }
 
-func (q *QuotaValidator) determineRequiredResources(allocatedResources corev1.ResourceList, shoot core.Shoot) (corev1.ResourceList, error) {
+func (q *QuotaValidator) determineRequiredResources(allocatedResources corev1.ResourceList, coreShoot core.Shoot) (corev1.ResourceList, error) {
+	shoot := gardencorev1beta1.Shoot{}
+	if err := gardencorev1beta1.Convert_core_Shoot_To_v1beta1_Shoot(&coreShoot, &shoot, nil); err != nil {
+		return nil, err
+	}
+
 	shootResources, err := q.getShootResources(shoot)
```

Comment:
The core Shoot is converted to version Shoot and then back inside `q.getShootResources`? Can we not pass the core Shoot directly?